src/plugins/htmlVideoPlayer/plugin.js

```diff
@@ -1051,9 +1051,12 @@ function tryRemoveElement(elem) {
             const avaliableFonts = [];
             const attachments = this._currentPlayOptions.mediaSource.MediaAttachments || [];
             const apiClient = ServerConnections.getApiClient(item);
-            attachments.map(function (i) {
+            attachments.filter(function (i) {
+                // ffmpeg fails to extract mjpeg attachment leading to HTTP 500 that stops playback
+                return i.Codec != 'mjpeg';
```

Comment:
This check can probably just be moved into the forEach to avoid looping over the array twice. Something like this:

```js
attachments.forEach(attachment => {
    if (attachment.Codec !== 'mjpeg') {
        availableFonts.push(apiClient.getUrl(attachment.DeliveryUrl));
    }
});